vendor/k8s.io/kube-openapi/pkg/handler/handler.go

```diff
@@ -77,58 +81,77 @@ func computeETag(data []byte) string {
 // NewOpenAPIService builds an OpenAPIService starting with the given spec.
 func NewOpenAPIService(spec *spec.Swagger) (*OpenAPIService, error) {
 	o := &OpenAPIService{}
+	// TODO(DangerOnTheRanger): is it safe to remove this UpdateSpec call?
 	if err := o.UpdateSpec(spec); err != nil {
 		return nil, err
 	}
 	return o, nil
 }
 
 func (o *OpenAPIService) getSwaggerBytes() ([]byte, string, time.Time) {
-	o.rwMutex.RLock()
```

Comment:
The 3 following `GetSwagger...` functions have very little value now, I would consider removing them.

Reply:
+1